consensus/clique/src/main/java/tech/pegasys/pantheon/consensus/clique/CliqueBlockInterface.java

```diff
@@ -58,14 +60,17 @@ public Address getProposerOfBlock(final BlockHeader header) {
   }
 
   public static BlockHeaderBuilder insertVoteToHeaderBuilder(
-      final BlockHeaderBuilder builder, final Optional<ValidatorVote> vote) {
-    if (vote.isPresent()) {
+      final BlockHeaderBuilder builder,
+      final EpochManager epochManager,
```

Comment:
There's a smell that the EpochManager is passed in from the caller - meaning the caller should/could have been able to make this decision to include the vote or not ...